README.md

```diff
@@ -376,6 +377,12 @@ Z = sqrt(tag<1>(X) * tag<1>(X) + tag<2>(Y) * tag<2>(Y));
 Here, the generated kernel will have one parameter for each of the vectors `X`
 and `Y`.
 
+### <a name="pointer-qualifiers"></a>Tagged terminals
+
+To optimise the code the programmer can use pointer qualifiers.
+At the moment the 'constant' qualifier is implemented. The qualifier will set the variable into a seperate memoryspace.
+This memoryspace is a separate cache and is as fast as the local memory. The cache size is vendor dependent. 
+
```

Comment:
But  if you do, you will need to move the `Pointer qualifiers` section after the `Permutations` section. Btw, the section heading is wrong in the current commit.